ament_cmake_clang_format/cmake/ament_cmake_clang_format_lint_hook.cmake

```diff
@@ -23,6 +23,10 @@ file(GLOB_RECURSE _source_files FOLLOW_SYMLINKS
   "*.hxx"
 )
 if(_source_files)
+  if(DEFINED ament_cmake_clang_format_CONFIG_FILE)
+    set(config_file_path ${ament_cmake_clang_format_CONFIG_FILE})
+  endif()
+
   message(STATUS "Added test 'clang_format' to check C / C++ code style")
-  ament_clang_format()
+  ament_clang_format(CONFIG_FILE ${config_file_path})
```

Comment:
This doesn't need to pass a `CONFIG_FILE` anymore.

Reply:
I was thinking that but was unsure if I should make the change or not - thanks